packages/widget-message-meet/src/components/activity-item-share-list/index.js

```diff
@@ -12,23 +12,45 @@ function ActivityItemShareList(props) {
   const {
     content,
     files,
+    isPending,
     onDownloadClick,
     share
   } = props;
 
   const items = files.map((file) => {
     if (file.image) {
-      const thumbnail = file.mimeType === `image/gif` ? share.getIn([`files`, file.url]) : share.getIn([`files`, file.image.url]);
+      let objectUrl;
+      let isFetching = true;
+      if (isPending) {
+        objectUrl = file.thumbnail;
+        isFetching = false;
+      }
+      else {
+        const thumbnail = file.mimeType === `image/gif` ? share.getIn([`files`, file.url]) : share.getIn([`files`, file.image.url]);
+        if (thumbnail) {
+          isFetching = thumbnail.get(`isFetching`);
+          objectUrl = thumbnail.get(`objectUrl`);
+        }
+      }
```

Comment:
Yea, might be best to do a constructor that gets the property correctly without the component having to know about mime types and locations.